common/ModIO.F90

```diff
@@ -519,6 +521,86 @@ subroutine ReadWallMesh(fn, wall)
 
   end subroutine ReadWallMesh
 
+
+!*********************************************************************
+  ! Given a cell (rbc) and a file name (fn), this writes the rbc's
+  ! plain coordinate values, celltype, and resolution (nlat & nlon) to the external file.
+  ! This file can later be reread into a later simulation to replicate the cell-shape of the inputted rbc.
+  ! This method was used after inducing an "UncurvedSickleSpheroid" (see ModRbc RBC_MakeUncurvedSickleSpheroid)
+  ! inside a flow, so that we can export the final curved sickle cell shape to a file for later usage.
+  subroutine ExportWriteRBC(fn, rbc)
+    character(*) :: fn
+    type(t_rbc) :: rbc
+
+    open(cell_unit, file=trim(fn), action='write')
+
+    write(cell_unit, *) rbc%nlat0, rbc%nlon0
+    write(cell_unit, *) rbc%nlat, rbc%nlon
+    write(cell_unit, *) rbc%celltype
+    write(cell_unit, *) rbc%x
+
+    close(cell_unit)
+  end subroutine ExportWriteRBC
+
+
+  ! Works in-tandem with ModIO ExportWriteRBC:
+  ! Given a cell (rbc), file name (fn), and optionally center (xc), import the data from the file
+  ! into the cell. The cell will be created with the resolution defined from the file data (and cannot be modified).
+  ! Used to import Sickle Cell model from the input file.
+  subroutine ImportReadRBC(fn, rbc, xc)
```

Comment:
shouldn't these subroutines `ImportReadRBC` and `ExportWriteRBC` be in `common/modIO.F90`? or somewhere else in the library, not carried around in specific input files.

Reply:
I'm sorry I didn't realize these are already in ModIO. Nevermind.